src/components/DashBody.js

```diff
@@ -0,0 +1,18 @@
+import React, { Component } from 'react'
+import DashWidget from './DashWidget'
+import '../styles/dash_body.css'
+
+class DashBody extends Component {
+  render() {
+    return (
+      <div className="dash-main">
+        <DashWidget/>
+        <div className="chat-icon">
```

Comment:
Another inline image to replace. Also, let's get the chat icon into its own component.